Cargo.toml

```diff
@@ -15,14 +15,14 @@ members = [
 ]
 
 [dependencies]
-secp256k1 = "0.26"
+secp256k1 = "0.28.0"
 rand_core = "0.6.4"
-ring = "0.16"
+ring = "0.17.5"
```

Comment:
`ring` is only used for SHA-256 and HMAC. If instead the `sha2` and `hmac` crates were used, the `ring` dependency could be dropped.